<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_fileManager</class>
 <widget class="QDialog" name="Dialog_fileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(163,181,191);</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222,233,239);</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>731</width>
     <height>321</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>729</width>
      <height>319</height>
     </rect>
    </property>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>731</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(222,233,239);</string>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>711</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_selectImage">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Selecionar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
